packages/account-sdk/src/store/store.ts

```diff
@@ -251,6 +271,20 @@ export const config = {
   },
 };
 
+export const userInfo = {
+  get: () => sdkstore.getState().userInfo,
+  set: (userInfo: Partial<UserInfo>) => {
+    sdkstore.setState((state) => ({
+      userInfo: { ...state.userInfo, ...userInfo },
```

Comment:
Shouldn't it replace the contents instead of adding?

```suggestion
      userInfo,
```